aten/src/ATen/native/nested/NestedTensorUtils.h

```diff
@@ -400,7 +400,15 @@ inline Tensor wrap_tensor_node(
     }
     options = flat_tensors[0].options().merge_in(options_);
     nt_buffer = at::cat(flat_tensors);
+
```

Comment:
Just for my own understanding, why don't we need to add calls to the nested tensor metadata like `_nested_tensor_size` and `_nested_tensor_stride` to [this bit](https://github.com/pytorch/pytorch/blob/master/torch/_subclasses/fake_tensor.py#L1080) ? I imagine these might need to be accessed in inductor

Reply:
My plan is to do the bare minimum in this PR to fakeify a NT and add whatever's needed for Inductor later on as we find it necessary. I imagine we'll need additional op meta registrations and such as well?